app/src/main/java/de/westnordost/streetcomplete/MainActivity.java

```diff
@@ -703,9 +705,7 @@ private boolean isQuestDetailsCurrentlyDisplayedFor(long questId, QuestGroup gro
 		f.setArguments(args);
 
 		android.app.FragmentTransaction ft = getFragmentManager().beginTransaction();
-		ft.setCustomAnimations(
-				R.animator.enter_from_bottom, R.animator.exit_to_bottom,
-				R.animator.enter_from_bottom, R.animator.exit_to_bottom);
+		ft.setCustomAnimations(R.animator.quest_answer_form_appear, R.animator.quest_answer_form_disappear);
```

Comment:
Did you test if the animation is still played on pop? (Pressing back)

Reply:
No it isn't played, because I removed this part as it plays the enter animation on exit first and then the exit animation...